conda_build/skeletons/cran.py

```diff
@@ -1165,6 +1166,23 @@ def skeletonize(in_packages, output_dir=".", output_suffix="", add_maintainer=No
         if 'R' not in dep_dict:
             dep_dict['R'] = ''
 
+        if 'rJava' in dep_dict:
+            print("Java is in dependencies")
```

Comment:
Will this handle transitive dependencies on rJava?

Reply:
also is it just debug text?

Reply:
Yes,it will add a skip on non Linux hosts for java.  Actually not sure if we want that within that commit, but logicial it belongs here IMHO.  I don't touch here the 'noarch' settings for now.  But logically the package isn't noarch, if we just build for a specific target,

Reply:
Removed that debug print